cypress/integration/instance/InstanceCreation.ts

```diff
@@ -44,10 +44,11 @@ describe("the 'Create a SE instance' Modal", () => {
               "This will be ready shortly."
             );
           });
+        cy.wait(150);
```

Comment:
I've investigated a bit the "detached" issue and I couldn't find other ways to fix it.

Doing two queries with a `0` wait also works, at least locally, but I don't know if it's any better.

```
cy.ouiaId("creating", "PF4/Button").should("be.visible");
cy.wait(0)
cy.ouiaId("creating", "PF4/Button").click();
```

It's clear that the table re-renders shortly after the data is displayed for the first time. You can even see a subtle movement of the columns when it happens. It's likely the table is doing some internal js calculation about the column layout to enable its responsive behavior.

Anyway, I think cypress should at least be able to re-query elements by itself when a detachment occurs. Fortunately it seems they are working on it, they recently merged a fix for this issue, it will be available with cypress 12.

Reply:
@kelvah thank you for investigating it. I have thought that the ResponsiveTable must re-render all elements but I did not catch when it happens. The addition of the small wait routine is without problem.

Do you think that this wait can be moved before cy.ouiaId("Instances list table", "PF4/Table")?

Reply:
@sjamboro AFAIK, to work, the wait should happen just before the click event.

Reply:
@vpellegrino thank you. I thought that the wait is connected with `It's clear that the table re-renders shortly after the data is displayed for the first time.` But probably Cypress require it from any reason.